### PR TITLE
chore(humanTask): update /API/bpm/humanTask GET request filters details

### PR DESCRIPTION
**Pull request by Shmayro**
Add missing filters :
- user_id
- show_assigned_to_others

closes [RUNTIME-1315](https://bonitasoft.atlassian.net/browse/RUNTIME-1315)

### CODE REVIEW

**Review by vbotnari12 — APPROVED** [review]
Looks good to me.

File: modules/api/pages/activities-tasks-flow-nodes-api.adoc [hunk 1]
```diff
@@ -582,11 +582,13 @@ xref:ROOT:rest-api-overview.adoc#resource_search[Standard search parameters] are
  ** o: name, priority, dueDate, state, processDefinitionId, processInstanceId,
 parentActivityInstanceId, assigneeId, parentContainerId, displayName, reachedStateDate
  ** s: search on any field that can be used to order results
- ** f: assigned_id, state, name, displayName, processDefinitionId, caseId, parentCaseId, rootCaseId
+ ** f: assigned_id, user_id, show_assigned_to_others, state, name, displayName, processDefinitionId, caseId, parentCaseId, rootCaseId
```
Comment (rbioteau) [inline modules/api/pages/activities-tasks-flow-nodes-api.adoc:585]: `user_id` should be part of a PR on an older branch.

**Review by rbioteau — CHANGES_REQUESTED** [review]

File: modules/api/pages/activities-tasks-flow-nodes-api.adoc (hunk 1 above)
Comment (vbotnari12) [reply modules/api/pages/activities-tasks-flow-nodes-api.adoc:585]: which one? :D

Comment (rbioteau) [reply modules/api/pages/activities-tasks-flow-nodes-api.adoc:585]: 2021.1

Comment (Shmayro) [reply modules/api/pages/activities-tasks-flow-nodes-api.adoc:585]: ok I will add new PR for that

Comment (Shmayro) [reply modules/api/pages/activities-tasks-flow-nodes-api.adoc:585]: do we need to do the same for `2021.2` and `2022.1`?

Comment (Shmayro) [reply modules/api/pages/activities-tasks-flow-nodes-api.adoc:585]: https://github.com/bonitasoft/bonita-doc/pull/2091

Comment (rbioteau) [reply modules/api/pages/activities-tasks-flow-nodes-api.adoc:585]: No, it will be merged

**Comment by rbioteau** [comment]
@Shmayro Can you update this PR please ?

**Pull request merged**